### PR TITLE
Updating the readme to point people to the HOWTO file

### PR DESCRIPTION
**Pull request by rcackerman**
I've updated the readme to be clearer to a semi-technical person (ie me).

I also separated the postgres part but am happy to put it back in.

### CODE REVIEW

**Comment by shakalee14** [comment]
Hey @rcackerman - can you resolve conflicts?

**Comment by rcackerman** [comment]
@shakalee14 Done! Sorry about that.

File: README.md [hunk 1]
```diff
@@ -14,41 +14,53 @@ The latest version is [2.0.0](https://github.com/MoveOnOrg/Spoke/tree/v2.0) (see
...
+### Filling out your `.env` file
+
+You now have a `.env` file to fill out. For more details on environment variables you can change, see the [environment variable reference](https://github.com/MoveOnOrg/Spoke/blob/main/docs/REFERENCE-environment_variables.md).
+
+There are some common environment variables you will want to adjust:
+
+1. To skip using the SMS provider (useful for development), set `DEFAULT_SERVICE=fakeservice`.
+2. Determine which database to use and set the necessary variables, listed in the [reference](https://github.com/MoveOnOrg/Spoke/blob/main/docs/REFERENCE-environment_variables.md).
+  - To use Postgres, [follow these instructions](https://github.com/MoveOnOrg/Spoke/blob/main/docs/HOWTO_USE_POSTGRESQL.md).
+3. Spoke uses [Auth0](https://auth0.com) by default. However, for development, there are a few ways to bypass authentication.
```
Comment (rcackerman) [inline README.md]: add `PASSPORT_STRATEGY=local` for dev

File: docs/HOWTO_USE_POSTGRESQL.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# How to set up Spoke with Postgresql
+
```
Comment (rcackerman) [inline docs/HOWTO_USE_POSTGRESQL.md:2]: The first time the app is run, the tables will be created.

**Review by schuyler1d — APPROVED** [review]
Thanks!

**Review by ibrand — APPROVED** [review]

**Comment by ibrand** [comment]
These are great. They feel really clear and helpful, thank you!

**Review by shakalee14 — APPROVED** [review]
Thanks so much! This is a lot more readable!

**Comment by shakalee14** [comment]
Merged with #1150

**Pull request closed without merging**